packages/sdk-browser/src/sdk-client.ts

```diff
@@ -273,21 +326,25 @@ export class TurnkeyBrowserClient extends TurnkeySDKClientBase {
       userId: userId!,
     });
 
+    // Ensure session and sessionExpiry are included in the object
+    const readWriteSessionResultWithSession = {
+      ...readWriteSessionResult,
+      session: readWriteSessionResult.credentialBundle,
```

Comment:
I wonder if we should rename `session` and `sessionExpiry` given how we renamed it earlier under the current User type. or are we intentionally naming it differently here to reduce confusion?

Reply:
well this is because the function takes the TSessionResponse type so I convert the object to that type before passing it in to saveSession
alternatively I could have the saveSession method take the type of `readWriteSessionResultWithSession` as is 
and differentiate between saving a read session vs a write session, what do you think?

Reply:
ohh I see, that makes sense. I'm fine with this approach in that case!